### ISSUE TITLE
Accessibility checks are different with the same url  (with Chromatic)

### ISSUE THREAD

**Issue by Lucasbeneston** [storybookjs/storybook#17571]
**Describe the bug**

**To Reproduce**

Step 1 : I click on the url and I see the accessibility checks  of the first component (button-inaccessible) with 
`1 violations | 16 assists | 0 incomplete`

Step 2 : I click on another component (ex: button-accessible) and I come back to the first one (button-inaccessible). 

Step 3 :  I have different values for the first component and all the other components have the same accessibility checks values : 
`0 violations | 1 assist | 0 incomplete`

**System**
Please paste the results of `npx sb@next info` here.

**Additional context**
